lib/classes/plugin_manager.php

```diff
@@ -1940,7 +1940,7 @@ public static function standard_plugins_list($type) {
             ),
 
             'qbank' => array(
-                 ''
+                'importquestions',
```

Comment:
can you please add an extra space after the comma? it will help the further merge of other plugins.